app/views/main.scala.html

```diff
@@ -23,6 +24,10 @@
 		<meta property="og:description" content="@description" />
 	}
 
+	@if(canonicalLink.isDefined) {
+		<link rel="canonical" href="@canonicalLink">
```

Comment:
Should this be `@canonicalLink.get`?

Reply:
I'm using the style from `description.isDefined` from above

Reply:
`isDefined` makes sure the `Option` is non-empty (not `None`). You still have to to a `get` or `getOrElse` to extract the content, no?

Reply:
It works without the get but I don't know why 
![image](https://user-images.githubusercontent.com/3072877/46727028-9b17c180-cc77-11e8-9002-eea1001a8ce4.png)
We get that and I'm not sure why it's not like `<link rel="canonical" href="Some(https://support.thegulocal.com/uk/subscribe/digital)">` or something

Anyway, let's map over them both as @LATaylor-guardian suggested. 😄